bridge/qb/shared/main.lua

```diff
@@ -85,12 +87,16 @@ qbShared.CommaValue = lib.math.groupdigits
 qbShared.RandomStr = function(length)
     if length <= 0 then return '' end
     local pattern = math.random(2) == 1 and 'a' or 'A'
+
+    ---@diagnostic disable-next-line: deprecated
     return qbShared.RandomStr(length - 1) .. lib.string.random(pattern)
```

Comment:
Instead of recursively calling this function, this should construct a string of '.' of length length, and only call lib.string.random() to avoid the deprecated call.